@radcortez is it normal that I cannot even successfully run the existing tests? I am running
```sh
./mvnw install -f extensions/jaeger/
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Quarkus - Jaeger                                                   [pom]
[INFO] Quarkus - Jaeger - Runtime                                         [jar]
[INFO] Quarkus - Jaeger - Deployment                                      [jar]
[INFO] 
[INFO] ------------------< io.quarkus:quarkus-jaeger-parent >------------------
[INFO] Building Quarkus - Jaeger 999-SNAPSHOT                             [1/3]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-scope) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-junit-scope) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java-version) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-no-runtime-deps) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ quarkus-jaeger-parent ---
[INFO] Executing: /bin/sh -c cd '/home/hak8fe/git/quarkus/extensions/jaeger' && 'git' 'rev-parse' '--verify' 'HEAD'
[INFO] Working directory: /home/hak8fe/git/quarkus/extensions/jaeger
[INFO] Storing buildNumber: e2209092818d8060cd84898f0e88261f74c2ada5 at timestamp: 1644933981128
[INFO] Storing buildScmBranch: fix_setting_of_sampler_manager_host_property
[INFO] 
[INFO] --- formatter-maven-plugin:2.17.1:format (default) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- impsort-maven-plugin:1.6.2:sort (sort-imports) @ quarkus-jaeger-parent ---
[INFO] Processed 0 files in 00:00.001 (Already Sorted: 0, Needed Sorting: 0)
[INFO] 
[INFO] --- maven-source-plugin:3.1.0:jar-no-fork (attach-sources) @ quarkus-jaeger-parent ---
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ quarkus-jaeger-parent ---
[INFO] Installing /home/hak8fe/git/quarkus/extensions/jaeger/pom.xml to /home/hak8fe/.m2/repository/io/quarkus/quarkus-jaeger-parent/999-SNAPSHOT/quarkus-jaeger-parent-999-SNAPSHOT.pom
[INFO] 
[INFO] ---------------------< io.quarkus:quarkus-jaeger >----------------------
[INFO] Building Quarkus - Jaeger - Runtime 999-SNAPSHOT                   [2/3]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-scope) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-junit-scope) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java-version) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-no-runtime-deps) @ quarkus-jaeger ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ quarkus-jaeger ---
[INFO] 
[INFO] --- formatter-maven-plugin:2.17.1:format (default) @ quarkus-jaeger ---
[INFO] Processed 16 files in 565ms (Formatted: 0, Unchanged: 16, Failed: 0, Readonly: 0)
[INFO] 
[INFO] --- impsort-maven-plugin:1.6.2:sort (sort-imports) @ quarkus-jaeger ---
[INFO] Processed 16 files in 00:00.023 (Already Sorted: 16, Needed Sorting: 0)
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ quarkus-jaeger ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] 
[INFO] --- quarkus-bootstrap-maven-plugin:999-SNAPSHOT:extension-descriptor (generate-extension-descriptor) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ quarkus-jaeger ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ quarkus-jaeger ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 0 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ quarkus-jaeger ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-jar-plugin:3.1.2:jar (default-jar) @ quarkus-jaeger ---
[INFO] Building jar: /home/hak8fe/git/quarkus/extensions/jaeger/runtime/target/quarkus-jaeger-999-SNAPSHOT.jar
[INFO] 
[INFO] --- maven-source-plugin:3.1.0:jar-no-fork (attach-sources) @ quarkus-jaeger ---
[INFO] 
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ quarkus-jaeger ---
[INFO] Installing /home/hak8fe/git/quarkus/extensions/jaeger/runtime/target/quarkus-jaeger-999-SNAPSHOT.jar to /home/hak8fe/.m2/repository/io/quarkus/quarkus-jaeger/999-SNAPSHOT/quarkus-jaeger-999-SNAPSHOT.jar
[INFO] Installing /home/hak8fe/git/quarkus/extensions/jaeger/runtime/pom.xml to /home/hak8fe/.m2/repository/io/quarkus/quarkus-jaeger/999-SNAPSHOT/quarkus-jaeger-999-SNAPSHOT.pom
[INFO] Installing /home/hak8fe/git/quarkus/extensions/jaeger/runtime/target/quarkus-jaeger-999-SNAPSHOT-sources.jar to /home/hak8fe/.m2/repository/io/quarkus/quarkus-jaeger/999-SNAPSHOT/quarkus-jaeger-999-SNAPSHOT-sources.jar
[INFO] 
[INFO] ----------------< io.quarkus:quarkus-jaeger-deployment >----------------
[INFO] Building Quarkus - Jaeger - Deployment 999-SNAPSHOT                [3/3]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-scope) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-junit-scope) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java-version) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-no-runtime-deps) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] --- formatter-maven-plugin:2.17.1:format (default) @ quarkus-jaeger-deployment ---
[INFO] Processed 8 files in 39ms (Formatted: 0, Unchanged: 8, Failed: 0, Readonly: 0)
[INFO] 
[INFO] --- impsort-maven-plugin:1.6.2:sort (sort-imports) @ quarkus-jaeger-deployment ---
[INFO] Processed 8 files in 00:00.001 (Already Sorted: 8, Needed Sorting: 0)
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ quarkus-jaeger-deployment ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/hak8fe/git/quarkus/extensions/jaeger/deployment/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ quarkus-jaeger-deployment ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ quarkus-jaeger-deployment ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ quarkus-jaeger-deployment ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ quarkus-jaeger-deployment ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.quarkus.jaeger.test.NestedMdcScopesTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.056 s - in io.quarkus.jaeger.test.NestedMdcScopesTest
[INFO] Running io.quarkus.jaeger.test.SamplerManagerConfigurationTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.426 s <<< FAILURE! - in io.quarkus.jaeger.test.SamplerManagerConfigurationTest
[ERROR] io.quarkus.jaeger.test.SamplerManagerConfigurationTest  Time elapsed: 1.426 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:613)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$10(ClassBasedTestDescriptor.java:381)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:381)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:205)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:80)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:330)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:566)
	... 42 more
Caused by: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:194)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:104)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:328)
	... 44 more
Caused by: java.lang.IllegalStateException: No root found for class io.quarkus.jaeger.runtime.ZipkinConfig
	at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadResult.requireRootObjectForClass(BuildTimeConfigurationReader.java:859)
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1134)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFromClass(ExtensionLoader.java:624)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:192)
	... 46 more

[INFO] Running io.quarkus.jaeger.test.JaegerMetricsTestCase
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.457 s <<< FAILURE! - in io.quarkus.jaeger.test.JaegerMetricsTestCase
[ERROR] io.quarkus.jaeger.test.JaegerMetricsTestCase  Time elapsed: 0.457 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:613)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$10(ClassBasedTestDescriptor.java:381)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:381)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:205)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:80)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:330)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:566)
	... 42 more
Caused by: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:194)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:104)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:328)
	... 44 more
Caused by: java.lang.IllegalStateException: No root found for class io.quarkus.jaeger.runtime.ZipkinConfig
	at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadResult.requireRootObjectForClass(BuildTimeConfigurationReader.java:859)
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1134)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFromClass(ExtensionLoader.java:624)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:192)
	... 46 more

[INFO] Running io.quarkus.jaeger.test.QuarkusJaegerTracerTest
[ERROR] OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.655 s - in io.quarkus.jaeger.test.QuarkusJaegerTracerTest
[INFO] Running io.quarkus.jaeger.test.CommaParseJaegerConfigurationTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.808 s <<< FAILURE! - in io.quarkus.jaeger.test.CommaParseJaegerConfigurationTest
[ERROR] io.quarkus.jaeger.test.CommaParseJaegerConfigurationTest  Time elapsed: 0.808 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:613)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$10(ClassBasedTestDescriptor.java:381)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:381)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:205)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:80)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:330)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.test.QuarkusUnitTest.beforeAll(QuarkusUnitTest.java:566)
	... 42 more
Caused by: java.lang.RuntimeException: Failed to load steps from class io.quarkus.jaeger.deployment.JaegerProcessor
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:194)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:104)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:328)
	... 44 more
Caused by: java.lang.IllegalStateException: No root found for class io.quarkus.jaeger.runtime.ZipkinConfig
	at io.quarkus.deployment.configuration.BuildTimeConfigurationReader$ReadResult.requireRootObjectForClass(BuildTimeConfigurationReader.java:859)
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1134)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFromClass(ExtensionLoader.java:624)
	at io.quarkus.deployment.ExtensionLoader.loadStepsFrom(ExtensionLoader.java:192)
	... 46 more

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   CommaParseJaegerConfigurationTest » Runtime java.lang.RuntimeException: java.l...
[ERROR]   JaegerMetricsTestCase » Runtime java.lang.RuntimeException: java.lang.RuntimeE...
[ERROR]   SamplerManagerConfigurationTest » Runtime java.lang.RuntimeException: java.lan...
[INFO] 
[ERROR] Tests run: 6, Failures: 0, Errors: 3, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Quarkus - Jaeger 999-SNAPSHOT:
[INFO] 
[INFO] Quarkus - Jaeger ................................... SUCCESS [  1.136 s]
[INFO] Quarkus - Jaeger - Runtime ......................... SUCCESS [  3.528 s]
[INFO] Quarkus - Jaeger - Deployment ...................... FAILURE [  7.458 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.685 s
[INFO] Finished at: 2022-02-15T15:06:32+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project quarkus-jaeger-deployment: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/hak8fe/git/quarkus/extensions/jaeger/deployment/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :quarkus-jaeger-deployment

```

--- a/extensions/jaeger/deployment/src/test/java/io/quarkus/jaeger/test/SamplerManagerConfigurationTest.java
+++ b/extensions/jaeger/deployment/src/test/java/io/quarkus/jaeger/test/SamplerManagerConfigurationTest.java
@@ -1,0 +1,31 @@
+package io.quarkus.jaeger.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.jaegertracing.Configuration.SamplerConfiguration;
+import io.quarkus.test.QuarkusUnitTest;
+
+/**
+ * Tests verifying configuration of a remote sampler manager.
+ *
+ */
+public class SamplerManagerConfigurationTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest TEST = new QuarkusUnitTest().withEmptyApplication()
+            .overrideConfigKey("quarkus.jaeger.service-name", "my-service")
+            .overrideConfigKey("quarkus.jaeger.sampler-manager-host-port", "my-jaeger-host:5778");
+
+    /**
+     * Verifies that the {@code JAEGER_SAMPLER_MANAGER_HOST_PORT} system property is set to the
+     * host name and port specified in the {@code quarkus.jaeger.sampler-manager-host-port} property.
+     */
+    @Test
+    void testSamplerManagerHostIsSetCorrectly() {
+        SamplerConfiguration config = SamplerConfiguration.fromEnv();
+        assertEquals("my-jaeger-host:5778", config.getManagerHostPort());
+    }
+}

--- a/extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentRecorder.java
+++ b/extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentRecorder.java
@@ -16,7 +16,7 @@ import io.quarkus.runtime.annotations.Recorder;
 @Recorder
 public class JaegerDeploymentRecorder {
     private static final Logger log = Logger.getLogger(JaegerDeploymentRecorder.class);
-    private static final Optional UNKNOWN_SERVICE_NAME = Optional.of("quarkus/unknown");
+    private static final Optional<String> UNKNOWN_SERVICE_NAME = Optional.of("quarkus/unknown");
     private static final QuarkusJaegerTracer quarkusTracer = new QuarkusJaegerTracer();
 
     public static String jaegerVersion;
@@ -81,7 +81,8 @@ public class JaegerDeploymentRecorder {
                 duration -> String.valueOf(duration.toMillis()));
         initTracerProperty("JAEGER_SAMPLER_TYPE", jaeger.samplerType, type -> type);
         initTracerProperty("JAEGER_SAMPLER_PARAM", jaeger.samplerParam, param -> NumberFormat.getInstance().format(param));
-        initTracerProperty("JAEGER_SAMPLER_MANAGER_HOST_PORT", jaeger.samplerManagerHostPort, hostPort -> hostPort.toString());
+        initTracerProperty("JAEGER_SAMPLER_MANAGER_HOST_PORT", jaeger.samplerManagerHostPort,
+                hostPort -> String.format("%s:%d", hostPort.getHostString(), hostPort.getPort()));
         initTracerProperty("JAEGER_SERVICE_NAME", jaeger.serviceName, name -> name);
         initTracerProperty("JAEGER_TAGS", jaeger.tags, tags -> tags.toString());
         initTracerProperty("JAEGER_PROPAGATION", jaeger.propagation, format -> format.toString());

